doc/source/knowledge_examples.rst

```diff
@@ -0,0 +1,3 @@
+=======================
+Knowledge Examples
+=======================
```

Comment:
is the inclusion of the example missing here?

Reply:
No, to have categories in the toc you need a page under which you can put other pages that's what this is for,.